src/main/java/io/getunleash/util/UnleashConfig.java

```diff
@@ -309,6 +327,7 @@ public static class Builder {
         private long fetchTogglesInterval = 10;
         private long sendMetricsInterval = 60;
         private boolean disableMetrics = false;
+        private UnleashFeatureFetcherFactory unleashFeatureFetcherFactory = HttpFeatureFetcher::new;
```

Comment:
@checketts Implement UnleashFeatureFetcherFactory and send it to the builder to use your custom Fetcher - `unleashFeatureFetcherFactory(..)` :)